docs/en_US/networkmorphismTuner.md

```diff
@@ -54,6 +54,52 @@ net = build_graph_from_json(RCV_CONFIG)
 nni.report_final_result(best_acc)
 ```
 
+If you want to save and load the best model, the following methods are recommended. 
```

Comment:
I recommend  we use bold font here...  "**load the best model**"

Reply:
👌